components/Buttons.tsx

```
@@ -0,0 +1,301 @@
+  return <Button {...props} variant="error" />
+}
+
+export function OutlineButton(props: BaseButtonProps) {
+  return <Button {...props} variant="outline" />
+}
+
+export function GhostButton(props: BaseButtonProps) {
+  return <Button {...props} variant="ghost" />
+}
+
+export function LinkButton(props: BaseButtonProps) {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'LinkButton'. Consider to delete it or rename it.

'LinkButton' is defined here:


(lint/suspicious/noRedeclare)